pkgs/by-name/mu/muse-sounds-manager/package.nix

```diff
@@ -55,24 +57,33 @@ stdenv.mkDerivation rec {
     openssl
   ];
 
-  unpackPhase = "dpkg -x $src .";
-
   installPhase = ''
     runHook preInstall
 
-    mkdir -p $out
-    mv usr/* opt $out/
-    substituteInPlace $out/bin/muse-sounds-manager --replace-fail /opt/ $out/opt/
+    mkdir -p $out $out/share/applications $out/share/icons
+    cp -p -R bin/ $out/
+    cp -p res/*.desktop $out/share/applications
+    cp -p -R res/icons $out/share
 
     runHook postInstall
   '';
 
+  postInstall = ''
+    ln -s ${xdg-utils}/bin/xdg-open $out/bin/open
+    wrapProgram $out/bin/muse-sounds-manager --prefix PATH : ${unzip}/bin
```

Comment:
```suggestion
    wrapProgram $out/bin/muse-sounds-manager \
      --prefix PATH : ${lib.makeBinpath [ unzip ]}
```

Reply:
Thank you for the review. The suggestion has been applied.